### ISSUE TITLE
AVT 1 & AVT 2 - React Inline Notification "Action" fails color contrast. 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3715]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React

## Detailed Description
Go to the React Inline Notification and run DAP and the Action button fails color contrast. See screenshot below:
<img width="1105" alt="Screen Shot 2019-08-12 at 2 08 57 PM" src="https://user-images.githubusercontent.com/21676914/62891299-f9378a00-bd0a-11e9-9d9e-7c219e61304d.png">
Then check it with the Color Contrast Anaylzer and it fails.
 See screenshot below:
<img width="1342" alt="Screen Shot 2019-08-12 at 2 07 21 PM" src="https://user-images.githubusercontent.com/21676914/62891156-980fb680-bd0a-11e9-911b-4709c065aa56.png">

**Comment by tw15egan** [comment]
@carbon-design-system/design what should this color value be updated to? It is currently using `$link-01`

**Comment by snidersd** [comment]
This issue is related to #3536

**Comment by dakahn** [comment]
@carbon-design-system/design bump

**Comment by dakahn** [comment]
Looks like it was fixed! Spooooky

**Comment by abbeyhrt** [comment]
I believe #3596 fixed the contrast issue by updating the token for the Action button

**Comment by snidersd** [comment]
@dakahn  Retested the Inline Notification with DAP. See and the same violation is still present so I reopened this issue:
<img width="1023" alt="Screen Shot 2019-11-01 at 3 09 52 PM" src="https://user-images.githubusercontent.com/21676914/68053345-b5c13c00-fcb9-11e9-8fa7-15183c6c2eae.png">

**Comment by laurenmrice** [comment]
The action link isn't calling the right token. The color of that link should be calling `inverse-link`

**Comment by aagonzales** [comment]
I believe this is fixed now through several updates.  `#78A9FF` on `#393939` passes at 4.9:1
http://react.carbondesignsystem.com/?path=/story/notifications--inline
